spec/models/manageiq/providers/amazon/aws_refresher_spec_common.rb

```diff
@@ -902,7 +876,7 @@ def assert_specific_vm_on_cloud_network_public_ip
       have_attributes(
         :config_version       => nil,
         :virtual_hw_version   => nil,
-        :guest_os             => options.inventory_object_refresh ? "linux_redhat" : nil,
+        :guest_os             => nil,
```

Comment:
This would be `"linux_redhat"` right?  If ` options.inventory_object_refresh` is true